roles/openshift_storage_glusterfs/tasks/glusterfs_deploy.yml

```diff
@@ -0,0 +1,108 @@
+---
+- fail:
+    msg: Only one GlusterFS nodeselector key pair should be provided
+  when: "openshift_storage_glusterfs_nodeselector.keys() | count > 1"
+
+- fail:
+    msg: There must be at least three GlusterFS nodes specified
+  when: "groups.oo_glusterfs_to_config | count < 3"
+
+- name: Delete pre-existing GlusterFS resources
```

Comment:
Is it necessary to remove resources, volumes, configs etc during another application of `openshift_storage_glusterfs`? I'd be worried about the impact these would have on an existing cluster where `playbooks/byo/config.yml` was reran to apply other configuration changes and glusterfs gets reset.